temporalio/workflow.py

```diff
@@ -4505,9 +4505,9 @@ async def wait(
     # but the "set" is changed out for a "list" and fixed up some typing/format
 
     if asyncio.isfuture(fs) or asyncio.iscoroutine(fs):
-        raise TypeError(f"expect a list of futures, not {type(fs).__name__}")
+        raise TypeError(f"Expect an iterable of Tasks/Futures, not {type(fs).__name__}")
     if not fs:
-        raise ValueError("Set of Tasks/Futures is empty.")
+        raise ValueError("Sequence of Tasks/Futures must not be empty.")
```

Comment:
This check doesn't actually make sense, since the input could be an empty generator, which is truthy, but will become empty when materialized as a list below. Leaving for now since this is copied from cPython.

Reply:
:+1: If they Python authors were better, it'd be below `fs = list(fs)` heh